### DISCUSSION TITLE
RRule support since Cron does not provide the same capabilities

### DISCUSSION THREAD

**Discussion by FebrevdMerwe** [quartznet/quartznet#2594] [Q&A]
Hi all

Is there a way in Quartz to create a trigger based on RRules? I need a way to schedule based on the RRule's similar to what Outlook and other calendar does. The problem I am facing is I don't see a way to create a single trigger to be able to give the functionality I need.

Take the following cases for example:
1. Every 5th day  (FREQ=DAILY;INTERVAL=5) - This can be done with the calendar trigger.
2. Every Second week Mon & Fri (FREQ=WEEKLY;INTERVAL=2;BYDAY=MO,FR)
3. Every 5th Month on the Last Friday (FREQ=MONTHLY;INTERVAL=3;BYDAY=-1FR)

For 2 I will need to create a separate calendar trigger for each day of the week since as far as I can see Quartz does not allow me to add multiple week days in a calendar trigger.

For 3 I have no idea how to do. 
The calendar trigger does not allow me to set the to trigger on the First, Second, Third or Last DayOfWeek
Cron trigger require me to specify the month of year the trigger should start. The issue with that is that the first month the trigger should fire will be different each year since 12 is not dividable by 5.

Is there a way in Quarts to create my own custom trigger or am I missing something?

**Comment by lahma** [comment]
There's https://github.com/quartznet/quartznet/pull/1276 , but a bit stale at the moment.
